src/test/Java/AnagramKataTest.java

```diff
@@ -0,0 +1,53 @@
+import org.assertj.core.api.Assertions;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.io.FileNotFoundException;
+
+public class AnagramKataTest {
+    AnagramKata anagramKata;
+@Before
+public void setUp() throws FileNotFoundException {
+ anagramKata = new AnagramKata("wordlist.txt");
+}
+
+    @Test
+    public void txtFileWordsWithNoDuplicates_AreSplitIntoSets() throws FileNotFoundException {
+        Assertions.assertThat(anagramKata.splitIntoSet("jakl")).containsKeys('j', 'k', 'l', 'a');
+    }
+
+    @Test
+    public void txtFileWordsWithDuplicates_AreSplitIntoSets() throws FileNotFoundException {
+        Assertions.assertThat(anagramKata.splitIntoSet("jakkl")).containsKeys('j', 'k', 'l', 'a').containsValue(2);
```

Comment:
I'm not sure this test is quite explicit enough, I'd want to know that the count for letter 'k' is 2 rather than for one of the letters in the list.